packages/esm-service-queues-app/src/send-patient-back-toqueue/add-patient-to-queue.scss

```diff
@@ -0,0 +1,91 @@
+@use '@carbon/styles/scss/spacing';
```

Comment:
Do we still need this file?

Reply:
I will do clean up @chibongho Thanks a lot